src/js/views/OperationsLog/ViewOperationsLog.jsx

```diff
@@ -91,6 +102,17 @@ function ViewOperationsLog({
     onDeleteClose()
   }
 
+  const duplicatesOpsLog = useCallback(() => {
+    const url = new URL('/ui/operations-log/create', globalState.baseURL)
+    Object.keys(cachedEntry)
```

Comment:
Because you're using the `cachedEntry` (which is when you click on an ops log from the table), it won't work if you were to share an ops log URL and click the duplicate button then. I would just switch this to `entry`. Also because this is already the reactive value in the dependency list.

Reply:
Fixed in [5b5585c](https://github.com/AWeber-Imbi/imbi-ui/pull/116/commits/5b5585ca5006d918d2056610426102b51e62e9cb) .... _it did work locally with a pasted link but I think that I got lucky because I used `entry` as a dependency_